config/config-defaults.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
FYI, instead of editing this defaults file, create a new file called `config-development.json` and add your values there.  That file is automatically excluded from git source.